presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionExtractor.java

```diff
@@ -63,6 +66,14 @@ protected Void visitPlan(PlanNode node, ImmutableList.Builder<Expression> contex
         }
 
         @Override
+        public Void visitAggregation(AggregationNode node, ImmutableList.Builder<Expression> context)
```

Comment:
Generally it is very easy to do this kind of mistake to forget to add plan node here.

Maybe instead of this we could make a `PlanNode` to be able to return owned `Expression`s? We had the same thing with `ChildrenReplacer` which used to be a `Visitor` and now each `PlanNode` is able to replace his children. 

What do you think?